### ISSUE TITLE
Could not save multiselect/select attribute options when Swatches modules disabled

### ISSUE THREAD

**Issue by kvydiuk** [magento/magento2#23326]
### Preconditions (*)

1. Magento 2.3.1
2. Disable swatches modules:
`php bin/magento module:disable Magento_SwatchesLayeredNavigation Magento_SwatchesGraphQl Magento_Swatches
php bin/magento setup:upgrade`
3. Applied patch:
https://github.com/magento/magento2/issues/20843#issuecomment-491445415
Commit:
https://github.com/magento/magento2/commit/1abadb5ddf82505c2048dac9312a5d58f21e5992

### Steps to reproduce (*)

1. Go to create product attribute page in backend.
2. Choose multiselect/select attribute type.
3. Add few options.
4. Save the attribute.

### Expected result (*)

1. Attribute options ARE saved

### Actual result (*)

1. Attribute options ARE NOT saved

**Comment by engcom-Bravo** [comment]
For 2.2-develop issue is reproduced as per issue #20843 where You weren't able to add options to the dropdown/multiselect attributes having that JS error. Obviously there was no Back-Port of the aforementioned commit (1abadb5ddf82505c2048dac9312a5d58f21e5992) to 2.2-develop. In 2.3-develop now You are able to add options but they do not remain after the attribute has been saved. No JS error occurs.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Bravo 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-17622`, `MC-17623` were created

**Issue Available**: @engcom-Bravo, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by kvydiuk** [comment]
Have temporary solution, if somebody interested in:
Magento/Catalog/view/adminhtml/templates/catalog/product/attribute/js.phtml
```php
<?php
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */

// @codingStandardsIgnoreFile

?>
<script>
require([
    "jquery",
    'Magento_Ui/js/modal/alert',
    'Magento_Ui/js/modal/prompt',
    'uiRegistry',
    "collapsable",
    "prototype"
], function(jQuery, alert, prompt, rg){

function toggleApplyVisibility(select) {
    if ($(select).value == 1) {
        $(select).next('select').removeClassName('no-display');
        $(select).next('select').removeClassName('ignore-validate');

    } else {
        $(select).next('select').addClassName('no-display');
        $(select).next('select').addClassName('ignore-validate');
        var options = $(select).next('select').options;
        for( var i=0; i < options.length; i++) {
            options[i].selected = false;
        }
    }
}
function getFrontTab() {
    if ($('product_attribute_tabs_front')) {
        return $('product_attribute_tabs_front').up('li');
    } else {
        return $('front_fieldset-wrapper');
    }
}

function initializeOptions() {
    var maxAttempts = 20,
        i = 0;

    var interval = setInterval(function () {
        if (rg.get('manage-options-panel') || i > maxAttempts) {
            var panel = jQuery('#manage-options-panel');
            if (panel.length) {
                panel.trigger('render');
            }
            clearInterval(interval);
        } else {
            i++;
        }
    }, 500);
}

function checkOptionsPanelVisibility(){
    if($('manage-options-panel')){
        var panelFieldset = $('manage-options-panel').up('.fieldset'),
            panel = jQuery('#manage-options-panel'),
            activePanelClass = 'selected-type-options';

        if($('frontend_input') && ($('frontend_input').value=='select' || $('frontend_input').value=='multiselect')){
            panelFieldset.show();
            jQuery(panelFieldset).addClass(activePanelClass);
            panel.addClass(activePanelClass);
            initializeOptions();
        }
        else {
            panelFieldset.hide();
            jQuery(panelFieldset).removeClass(activePanelClass);
            panel.removeClass(activePanelClass);
        }
    }
}
...
```

**Comment by magento-engcom-team** [comment]
Hi @kirillvidyuk. 

Thank you for your report and collaboration!

The issue was fixed by Magento team. The fix was delivered into `magento/magento2:2.3-develop` branch(es).
Related commit(s):
 - [084248165504331c7eb8b62407d83115226fcbd2](https://github.com/magento/magento2/commit/084248165504331c7eb8b62407d83115226fcbd2)

The fix will be available with the upcoming `2.3.3` release.
